app/assets/javascripts/components/authentication/Register.jsx

```diff
@@ -54,9 +54,10 @@ class Register extends React.Component {
       user: user,
     }
 
-    Requester.post('/api/users/show', params).then((data) => {
+    Requester.post('/api/users/', params).then((data) => {
       window.location.href = '/';
     }).catch((data) => {
+      console.log(data)
```

Comment:
you can do `console.error(data)` here in order to display an error box in the console